chain/hyperliquid/tx_input/tx_input.go

```diff
@@ -0,0 +1,74 @@
+package tx_input
+
+import (
+	"time"
+
+	xc "github.com/cordialsys/crosschain"
+	"github.com/cordialsys/crosschain/factory/drivers/registry"
+)
+
+const UsdcTokenId = "USDC:0x6d1e7cde53ba9467b783cb7c530ce054"
+const SafetyTimeoutMargin = (48 * time.Hour)
+
+// TxInput for Hyperliquid
+type TxInput struct {
+	xc.TxInputEnvelope
+	TransactionTime int64 `json:"transaction_time"`
```

Comment:
let's use `time.Time` which will use rfc3339 encoding by default.  That way it's more clear what units are being used where.